src/Vocup/MainForm.cs

```diff
@@ -356,6 +357,38 @@ private async void TsmiUpdate_Click(object sender, EventArgs e)
     private void TsmiSaveAs_Click(object sender, EventArgs e) => SaveFile(true);
     private void TsbSave_Click(object sender, EventArgs e) => SaveFile(false);
 
+    private void TsmiShare_Click(object sender, EventArgs e)
```

Comment:
@jdelange22 I created a `ToolStripMenuItem` to share a book without practice results. It work similar to save as, but does not save practice results and instead of opening the newly saved file in Vocup, it opens the location in Windows Explorer.

I played around with [`DataTransferManager`](https://learn.microsoft.com/de-de/uwp/api/windows.applicationmodel.datatransfer.datatransfermanager?view=winrt-22621) from WinRT but was not able to get it to work with Windows Forms and .NET 8.0. Even if it would work, this approach is not supported by many apps except Outlook and the WhatsApp UWP app.

Reply:
Nice one!